entrypoint.sh

```diff
@@ -61,6 +63,7 @@ fi
 # if not, launch it, but don't deploy yet
 if ! flyctl status --app "$APP"; then
   flyctl apps create "$APP" --org "$ORG"
+  fly scale vm "$VM" --memory "$VM_MEMORY" --app "$APP"
```

Comment:
What do you think about changing `fly` to `flyctl` to maintain consistency? I know this may be unnecessary, since they both work the same.
Or, changing all instances of `flyctl` to `fly`.